test_runner/batch_others/test_tenant_relocation.py

```diff
@@ -115,14 +116,24 @@ def load(pg: Postgres, stop_event: threading.Event, load_ok_event: threading.Eve
     log.info('load thread stopped')
 
 
-def populate_branch(pg: Postgres, create_table: bool,
-                    expected_sum: Optional[int]) -> Tuple[UUID, int]:
+def populate_branch(
+    pg: Postgres,
+    tenant_id: UUID,
+    ps_http: NeonPageserverHttpClient,
+    create_table: bool,
+    expected_sum: Optional[int],
+) -> Tuple[UUID, int]:
     # insert some data
     with pg_cur(pg) as cur:
         cur.execute("SHOW neon.timeline_id")
         timeline_id = UUID(cur.fetchone()[0])
         log.info("timeline to relocate %s", timeline_id.hex)
 
+        cur.execute("SELECT pg_current_wal_flush_lsn()")
+        log.info("pg_current_wal_flush_lsn() %s", lsn_from_hex(cur.fetchone()[0]))
+        log.info("timeline detail %s",
+                 ps_http.timeline_detail(tenant_id=tenant_id, timeline_id=timeline_id))
```

Comment:
Does it really logs as expected? I'd expect f-strings or `"timeline detail {}".format(.....)` instead.

Reply:
Yeah, it works :) This method works better for integration with various logging middleware. For example if you use Sentry it will be able do dedup events by taking into account only format string, not the parameters. If parameters are pre filled like you proposed then it wont be able to do so and will create separate tickets for every combination of user_id/request_id/whatever that came with the event. It is highly unlikely that we'll need to use something like Sentry in tests, but this is the way that I used to cook python logging just to adhere to least surprise principle